content/ch-algorithms/quantum-counting.ipynb

```diff
@@ -305,7 +305,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_format = 'svg'"
```

Comment:
Thanks for rendering as SVG! Looks much better on the web. Can you please remove this line though? It should be set in the config and not in the notebook (to keep things simpler for readers).

Reply:
Ah, I forgot to remove it. Thanks for pointing out!